Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -33,11 +33,13 @@ WarpX::LoadBalance ()
 
     // By default, do not do a redistribute; this toggles to true if RemakeLevel
     // is called for any level
-    int doLoadBalance = 0;
+    int loadBalancedAnyLevel = false;
```

Comment:
Haha, I meant you could change the types if the Bcast overloads support it :)
```suggestion
    bool loadBalancedAnyLevel = false;
```

Reply:
The BCast overloads don't support it, however, I do think `int var = false` is a little bit clearer as to the intent of the variables.